File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    answer = 1
    dic = {}
    for elem in clothes:
        if elem[1] in dic.keys():
            dic[elem[1]] += 1
        else:
            dic[elem[1]] = 1
    for value in dic.values():
        answer *= (value + 1)
    return (answer - 1)

File: 프로그래머스/2/42578. 의상/의상.java
import java.util.HashMap;

class Solution {
    
    public int solution(String[][] clothes) {
        int answer = 1;
        HashMap<String, Integer> h = new HashMap<>();
        for(int i = 0; i < clothes.length; i++) {
          h.put(clothes[i][1], h.getOrDefault(clothes[i][1], 0) + 1);
        }
        for(int elem : h.values()) {
          answer *= (elem + 1);
        }
        return (answer - 1);
    }
}